Sources/CMakeLists.txt

```diff
@@ -110,6 +113,7 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     add_definitions(-Wno-switch)
     add_definitions(-Wno-tautological-undefined-compare)
     add_definitions(-Wno-c++11-compat-deprecated-writable-strings)
+    add_definitions(-Wno-logical-op-parentheses)
```

Comment:
the problem with those are that the code is /probably/ correct as it is, and if it's not then adding incorrect braces makes the issue even harder to find..  
still may be worthwhile to fix those warnings once everything seems stable :)